addons/udes_sale_stock/models/stock_picking.py

```diff
@@ -33,3 +33,22 @@ def add_more_pickings_to_reserve(self, picking_type):
                 lambda p: p.picking_type_id.u_full_sale_reservation and p.state == "confirmed"
             )
         return pickings
+
+    def get_pallet_move_details_extra_info(self, result, package, container_id=None):
+        """Extend to include information from the sale"""
+        res = super().get_pallet_move_details_extra_info(result, package, container_id=container_id)
+        # If the picking type configuration is set,
```

Comment:
i will remove this comment to say;
`# Replace the picking name with the sales order name.`

as the check no longer occurrs in this function

Reply:
eb2dcff61ef531de65a1b457b6282c98338ec20c